README.md

```diff
@@ -10,4 +10,396 @@ TODO: description
 
 ## API Routes
 
-TODO: document API routes
+### Shared Account
+
+#### `POST /api/accounts` - Create an new shared account
+
+**Body**
+
+- `name` _{string}_ - The account owner's name
+- `username` _{string}_ - The user's username
+- `password` _{string}_ - The user's password
+
+**Returns**
+
+- A success message
+- The created shared account's details
+
+**Throws**
+
+- `403` if there is a user already logged in
+- `400` if `name` is empty or missing
+- `400` if `username` or `password` is not in correct format or missing
+- `409` if `username` is already in use
+
+#### `PATCH /api/accounts` - Update a shared account's details
+
+**Body**
+
+- `name` _{string}_ - The new name
```

Comment:
Similar to above, if we do edit the account owner's name format, this will change as well